packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -804,29 +832,35 @@ void main() {
     await tester.tap(find.byType(DropdownMenu<TestMenu>));
     await tester.pump();
 
+    final bool isMobile = themeData.platform == TargetPlatform.android || themeData.platform == TargetPlatform.iOS || themeData.platform == TargetPlatform.fuchsia;
+    int expectedCount = isMobile ? 0 : 1;
```

Comment:
isMobile should really be a late final that's initialized with a switch. per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values

Reply:
Fixed. Thanks!